app/code/Magento/Catalog/Test/Unit/Model/Category/FileInfoTest.php

```diff
@@ -116,6 +116,9 @@ function ($arg) use ($baseDirectory, $pubDirectory) {
         $this->pubDirectory->method('getAbsolutePath')
             ->willReturn('/a/b/c/pub/');
 
+        $this->store->method('getBaseUrl')
+            ->willReturn('');
```

Comment:
I think we should add there some better examples, like
```suggestion
            ->willReturn('https://example.com/');
```

Reply:
Hello @ihor-sviziev!
Thanks for your suggestions.
This fix only adds a compatibility with PHP 8.1 and doesn't change any logic. This changes adds compatibility to [FileInfo.php](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/Model/Category/FileInfo.php#L226)
But I think you are right. We can set some base url because I think it's always not empty.